shuup/core/models/_suppliers.py

```diff
@@ -169,23 +202,30 @@ def get_orderability_errors(self, shop_product, quantity, customer):
         :type contect: shuup.core.models.Contact
         :rtype: iterable[ValidationError]
         """
-        return self.module.get_orderability_errors(shop_product=shop_product, quantity=quantity, customer=customer)
+        return self.modules[
```

Comment:
from where is `self.modules` coming from?

Reply:
The property method from the supplier interface.